.gitignore

```diff
@@ -1,6 +1,6 @@
 .DS_Store
 .env
-broadcast/DeployLocal.s.sol/
+broadcast/
```

Comment:
I've noted in the PR description that numerous deployments across various chains have executed the deploy script multiple times (2 to 4 times) because of issues with RPC nonce or transactions dropped due to low gas. This has resulted in cluttered logs.